### DISCUSSION TITLE
Migration from Hypercontext/linkifyjs to nfrasser/linkifyjs

### DISCUSSION THREAD

**Discussion by nozaq** [nfrasser/linkifyjs#496] [Q&A] [answered]
Hello,

I noticed that this repository has recently been moved from https://github.com/Hypercontext/linkifyjs to https://github.com/nfrasser/linkifyjs, but couldn't find any relevant discussion in Issues and Discussions.
Also from the commit history, the new owner account(@nfrasser) seems to have the same name as the past contributor(maintainer, probably?), but the user account itself is different.

Is there any way to verify this migration is legitimate? Sorry for asking such oversuspicious question and if I missed anything obvious here 🙏🏼

**Comment by nfrasser** [comment, marked as answer]
Hi @nozaq, thanks for your question. Yes, this migration is legitimate. Hypercontext has unfortunately gone out of business. However, I remain the primary maintainer of linkify as I have been since it started, and so the company has transferred the project to my account. 

I'm not sure if my verification is enough to convince you - would an update to the README be helpful? Perhaps @grahammccarthy, one of @Hypercontext's owners, can verify?

**Reply by grahammccarthy** [reply]
Hi all! Whoops, I must have missed this. Yes, unfortunately Hypercontext has shut down but we didn't want to see such an awesome project like Linkify.js end with us.  Nick has put in a ton of hard work over the years and will be an excellent steward of the project moving forward!

Thanks @nfrasser !!

**Comment by nozaq** [comment]
@nfrasser Thank you for your reply. I agree that updating README will definitely help people understand the situation. I believe the following two facts provide enough evidence, so no additional verification is necessary for me.

- [Hypercontext/linkifyjs](https://github.com/Hypercontext/linkifyjs,) is redirecting to [nfrasser/linkifyjs](https://github.com/nfrasser/linkifyjs), which indicates @Hypercontext approved this migration.
- Your account(@nfrasser) have been involved in many issues in this repository since a long time ago. Sorry I only checked the commit history before opening this thread, but I realize your account is the one who even opened [the first issue](https://github.com/nfrasser/linkifyjs/issues/1) in this repository 😓 

That being said, I really appreciate that you keep maintaining this project, it helps our project a lot!
